repos/fdbt-site/src/design/modules/_components.scss

```diff
@@ -32,106 +27,115 @@
             transform: translateY(-50%);
             line-height: normal;
         }
-
         &__input {
             padding-left: 22px;
         }
     }
 }
-
 .flex-container {
     display: flex;
     align-items: flex-end;
 }
-
 .padding {
     padding-left: 15px;
 }
-
 .pagination {
     padding: 0;
 }
-
 .pagination__item {
     display: inline-block;
     list-style: none;
 }
-
 .pagination__link {
     display: block;
     padding: 5px 10px;
     text-decoration: none;
-
     &:hover,
     &:focus {
         background: #f8f8f8;
         outline: 3px solid #ffbf47;
     }
-
     &.current {
         color: #0b0c0c;
         font-weight: 700;
         border: none;
         pointer-events: none;
         cursor: default;
-
         &:hover,
         &:focus {
             color: #0b0c0c;
             background: none;
         }
     }
 }
-
 .pagination__summary {
     font-family: 'Inter UI', sans-serif;
     -webkit-font-smoothing: antialiased;
     padding: 8px 0;
-
     @media (min-width: 642px) {
         float: right;
     }
 }
-
 .information_box__success {
     display: block;
     padding: 20px;
     margin-bottom: 50px;
     border: 5px solid;
     border-color: govuk-colour('green');
 }
-
 .information_box__warning {
     display: block;
     padding: 20px;
     margin-bottom: 50px;
     border: 5px solid;
     border-color: govuk-colour('red');
 }
-
 .expiry-selector-units {
     border-left: 0;
 }
-
 .expiry-selector-wrapper {
     display: flex;
 }
-
 .new-tag {
     background-color: #cce2d9;
     color: #055934;
     margin-right: 10px;
 }
-
 .button-link {
     color: govuk-colour('blue');
     background-color: govuk-colour('white');
     padding: 0;
     border: 0;
     text-decoration: underline;
     cursor: pointer;
-
     &:hover {
         color: govuk-shade(govuk-colour('red'), 20%);
     }
 }
+.selectedOperators-header-color {
+    background-color: #dddde2;
```

Comment:
should we have this formatted like before with a gap between each class - seems to have been removed

Reply:
that was @RobCattonKPMG  suggestion to remove gaps